### PR TITLE
Fix wrong unit bitrate

### PR DESCRIPTION
**Pull request by lherman-cs**

### CODE REVIEW

File: pkg/codec/x264/x264.go [hunk 1]
```diff
@@ -67,6 +67,10 @@ func newEncoder(r video.Reader, p prop.Media, params Params) (codec.ReadCloser,
 		params.KeyFrameInterval = 60
 	}
 
+	// Convert from bit/s to kbit/s because x264 uses kbit/s instead.
+	// Reference: https://code.videolan.org/videolan/x264/-/blob/7923c5818b50a3d8816eed222a7c43b418a73b36/encoder/encoder.c#L657
```
Comment (at-wat) [inline pkg/codec/x264/x264.go:71]: This link seems not related to the bit-rate?

Comment (lherman-cs) [reply pkg/codec/x264/x264.go:71]: Oops, it's a wrong file
